src/routes/login/+page.svelte

```diff
@@ -1 +1,98 @@
-<h1>Login</h1>
+<script lang="ts">
+	let email = '';
+	let password = '';
+	let isLoading = false;
+	let isSuccess = false;
+
+	let errors = {
+		email: undefined,
+		password: undefined,
+	};
+
+	const handleSubmit = () => {
+		console.log('Email:', email);
+		console.log('Password: ', password);
+	};
+</script>
+
+<form on:submit|preventDefault={handleSubmit}>
```

Comment:
I thought we modified this to be handled by the server and redirect to the correct page?